src/ApiEntity/OrderItem.php

```diff
@@ -51,52 +45,4 @@ public function setOrderId(string $value): self
 
         return $this;
     }
-
```

Comment:
was getters and setters to those 2 extra fields accidentally removed, or github doesn't show diff properly again?